### PR TITLE
feat(alerts): implement muting rules

### PR DESCRIPTION
**Pull request by zlesnr**

### CODE REVIEW

File: pkg/alerts/muting_rules.go [hunk 1]
```diff
@@ -0,0 +1,175 @@
+package alerts
+
+// MutingRule represents the alert suppression mechanism in the Alerts API.
+type MutingRule struct {
+	ID          int                      `json:"id,string,omitempty"`
+	AccountID   int                      `json:"accountId,string,omitempty"`
```
Comment (sanderblue) [inline pkg/alerts/muting_rules.go]: Looks like `accountId ` is returned as type `Int` in the NerdGraph response. Do we want to unmarshal this to a string per the current JSON tag? Seems like we would want to unmarshal it to an `int`.

File: pkg/alerts/muting_rules.go [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+	alertsMutingRulesCreate = `mutation CreateRule($accountID: Int!, $rule: AlertsMutingRuleInput!) {
+		alertsMutingRuleCreate(accountId: $accountID, rule: $rule) {
+			id
+		}
+	}`
+
+	alertsMutingRulesUpdate = `mutation UpdateRule($accountID: Int!, $ruleID: ID!, $rule: AlertsMutingRuleUpdateInput!) {
+		alertsMutingRuleUpdate(accountId: $accountID, id: $ruleID, rule: $rule) {
+			name
```
Comment (sanderblue) [inline pkg/alerts/muting_rules.go]: Thoughts on returning other fields here?

File: pkg/alerts/muting_rules.go (hunk 1 above)
Comment (zlesnr) [reply pkg/alerts/muting_rules.go]: Good catch.  I'm surprised it worked at all.

File: pkg/alerts/muting_rules.go (hunk 2 above)
Comment (zlesnr) [reply pkg/alerts/muting_rules.go]: Included description and enabled.

**Pull request merged**